.jsdoc.js

```diff
@@ -39,7 +39,7 @@ module.exports = {
     copyright: 'Copyright 2018 Google, LLC.',
     includeDate: false,
     sourceFiles: false,
-    systemName: '@google-cloud/datacatalog',
+    systemName: 'datacatalog',
```

Comment:
@bcoe I'm guessing this isn't right?

Reply:
aye, looks like we need to update the package name in the readme